templates/web/base/report/_inspect.html

```diff
@@ -16,6 +16,9 @@ <h2 class="inspect-form-heading">[% loc('Inspect report') %]</h2>
         <p>
           <strong>[% loc('Report ID:') %]</strong>
           <span class="js-report-id">[% problem.id %]</span>
+          [% IF c.user_exists AND c.cobrand.admin_allow_user(c.user) AND c.user.has_permission_to('report_edit', problem.bodies_str_ids) %]
```

Comment:
Can someone have report edit permission and not have admin_allow_user? Guess can't hurt.

Reply:
Right now I don't think that's possible, but as you say it can't hurt to check.